website/blog/2023-10-26-release-1.5.md

```diff
@@ -0,0 +1,219 @@
+---
+title: Release Notes - Podman Desktop 1.5
+description: Podman Desktop 1.5 has been released!
+slug: podman-desktop-release-1.5
+authors: duffy
+tags: [podman-desktop, release, kubernetes, openshift, onboarding]
+hide_table_of_contents: false
+---
+
+import ReactPlayer from 'react-player'
+
+Podman Desktop 1.5 Release! 🎉
+
+With this release of Podman Desktop, we're introducing a new onboarding feature that we hope will earn your seal of approval! But wait... there's so much more!
```

Comment:
How did none of us use this before?? Awesome intro.

Reply:
I actually went back thru old posts to make sure we hadn't :seal: 🤣